apps/mobile/src/components/ContextMenu/ContextMenu.js

```diff
@@ -0,0 +1,217 @@
+import React, { useMemo } from 'react'
+import { View, Text, TouchableOpacity, ScrollView } from 'react-native'
+import { useTranslation } from 'react-i18next'
+import { useNavigation } from '@react-navigation/native'
+import { PUBLIC_CONTEXT_SLUG, MY_CONTEXT_SLUG, ALL_GROUPS_CONTEXT_SLUG } from '@hylo/shared'
+import { widgetUrl as makeWidgetUrl } from 'util/navigation'
+import FastImage from 'react-native-fast-image'
+import useCurrentUser from '@hylo/hooks/useCurrentUser'
+import useCurrentGroup, { useCurrentGroupSlug } from '@hylo/hooks/useCurrentGroup'
+import useRouteParams from 'hooks/useRouteParams'
+import ContextWidgetPresenter, { orderContextWidgetsForContextMenu } from '@hylo/presenters/ContextWidgetPresenter'
+import useGetContextWidgetChildren from '@hylo/hooks/useGetContextWidgetChildren'
+import useHasResponsibility, { RESP_ADD_MEMBERS, RESP_ADMINISTRATION } from '@hylo/hooks/useHasResponsibility'
+import useLogout from 'hooks/useLogout'
+import WidgetIconResolver from 'components/WidgetIconResolver'
+import GroupMenuHeader from 'components/GroupMenuHeader'
+import { openURL } from 'hooks/useOpenURL'
+import { DEFAULT_APP_HOST } from 'navigation/linking'
+import { URL } from 'react-native-url-polyfill'
+import Loading from 'components/Loading'
+
+export default function ContextMenu () {
+  const { myHome, groupSlug } = useRouteParams()
+  const [{ currentGroup, fetching }] = useCurrentGroup({ setToGroupSlug: groupSlug })
+  const currentGroupSlug = useCurrentGroupSlug()
+  const navigation = useNavigation()
+  const { t } = useTranslation()
+  const contextWidgets = currentGroup?.contextWidgets || []
+  const orderedWidgets = useMemo(() =>
+    orderContextWidgetsForContextMenu(contextWidgets.map(widget => ContextWidgetPresenter(widget, { t }))), [contextWidgets, currentGroup, myHome]
+  )
+
+  if (fetching && currentGroup) return <Loading />
+  if (!currentGroup) return null
+
+  const isPublic = currentGroup.slug === PUBLIC_CONTEXT_SLUG
+  const isMyContext = currentGroup.slug === MY_CONTEXT_SLUG
+  const isAllContext = currentGroup.slug === ALL_GROUPS_CONTEXT_SLUG
+
+  return (
+    <View className='flex-1 bg-background'>
+      <ContextHeader
+        group={currentGroup}
+      />
+      <ContextWidgetList
+        contextWidgets={orderedWidgets}
+        groupSlug={currentGroup.slug}
+        rootPath={`/groups/${currentGroup.slug}`}
+      />
+      {/* Add the All Views widget at the bottom, matching web behavior */}
+      {(!isMyContext && !isPublic && !isAllContext) && (
```

Comment:
Just make this `!currentGroup.isContextGroup` and do away with the `const`s above? I realize that there will likely be use of the differences between the three coming in here soon, but would rather keep that out until we see what emerges. We should be able to handle this simply by an attribute on a presented group, like group.type? OR whatever the right naming is for that as type may already be used for something else? Ideally we are switching here based on a presented group's attributes already resolved, with "type" being the most primitive.